docs/shared/esg_grid.yaml

```diff
@@ -0,0 +1,21 @@
+esg_grid:
+  execution:
+    batchargs:
+      nodes: 1
+      stdout: /path/to/runscript.out
+      walltime: "08:00:00"
+    executable: /path/esg_grid
+  namelist:
+    update_values:
```

Comment:
Looking at this again, I don't have a `base_file` entry here and I think it should be added based on our previous conversations.

Reply:
Since it's used for documentation, I agree with you: Showing as complete an example as possible probably helps readers.